### DISCUSSION TITLE
Registration Default Value and Deletion of Users

### DISCUSSION THREAD

**Discussion by choseh** [pretix/pretix#2571] [Ideas] [answered]
Hello,

I have noticed that several users have registered on my pretix instance as actual pretix users (/control) instead of registering with an organizer. I have not yet found the option to delete them from the backend.

a) I think it's better to default to registration=off to prevent this kind of behavior and stick to email invites for new team members/admins
b) How can I delete those users? I assume it's hidden behind the "Anonymize" option and possible after this, this should be a little more clear IMHO

Thanks,
Alex

**Comment by pc-coholic** [comment, marked as answer]
> b) How can I delete those users? I assume it's hidden behind the "Anonymize" option and possible after this, this should be a little more clear IMHO

Anonymizing a user will only remove all identifying information like the email-address and the name. But there is no way to completely delete a user, as pretix is referencing the user-object in a lot of places (for example: Logging when something has been changed, etc.). As such, deleting a user would yield broken references.

**Reply by choseh** [reply]
Okay, I see. Thanks for the explanation.

**Comment by raphaelm** [comment]
I think I'm +1 on changing the default here
